spec/controllers/insights_controller_spec.rb

```diff
@@ -0,0 +1,2 @@
+RSpec.describe InsightsController do
+end
```

Comment:
This test file doesn't appear to be testing anything?

Reply:
What this should be testing?

Reply:
After me and Margo talked I will delete this empty file.

Reply:
If there's no reason to test the controller, do we really need an empty test file?

Reply:
I deleted ♻️